.github/workflows/rust.yml

```diff
@@ -11,11 +11,13 @@ jobs:
     - uses: actions/checkout@v1
     - name: Build
       run: cargo build --verbose
-    - name: Run tests
-      run: cargo test --verbose
+    - name: Run unit tests
+      run: cargo test --locked  --no-fail-fast --lib
+    - name: Run int tests
+      run: cargo test --test lib --no-fail-fast --features "json"
```

Comment:
Why is there a `lib` here? I am a bit confused since you're using `lib` here and `--lib` flag on `.travis.yml`

Reply:
the lib file in the `test/` dir